src/main/java/frc/robot/Controls.java

```diff
@@ -372,6 +380,33 @@ private void configureSuperStructureBindings() {
                   .withName("Automatic Intake"));
     }
 
+    if (sensors.armSensor != null) {
```

Comment:
Extend the if (sensors.armSensor != null) {} from line 365 above.  No need to add another check/nesting.